pkgs/applications/emulators/retroarch/cores.nix

```diff
@@ -732,7 +732,8 @@ in
     description = "Amiga emulator based on WinUAE";
     license = lib.licenses.gpl2Only;
     makefile = "Makefile";
-      postPatch = ''
+    # Delete this patch after https://github.com/libretro/libretro-uae/pull/529 is merged
+    postPatch = ''
```

Comment:
Why not use `fetchpatch` instead and get the patch from upstream?

Reply:
Didn't find `pull request url + .patch` the first time, and didn't want to include the patch in the source tree. Thank you for suggestion